Northstar.Custom/mod/scripts/vscripts/gamemodes/sh_gamemode_fw_custom.nut

```diff
@@ -74,6 +78,8 @@ void function FWOnRegisteringNetworkVars()
 	if ( GAMETYPE != FORT_WAR )
 		return
 	
+	Remote_RegisterFunction( "ServerCallback_FW_NotifyNeedsEnterEnemyArea" )
```

Comment:
From testing, this will cause compatibility issues with older clients. Not sure if when merging we wanna remove that line and only add the fix on version gate or if we wanna just push it out and ask clients to update ^^

(Keep it for now)

Reply:
yes this pr breaks older clients, it's a update with new client functions